MoltBrowser patch for components/browsing_data_strings.grdp
Upstream base: chromium/src@d331de35ca8ab1358c7a17e4bd41627d55eaef51 (156.0.8078.74)
Generated from src/components/browsing_data_strings.grdp by tools/chromium/overlay_to_patches.py.
src/ is still the source of truth: edit the file there and
re-run the generator (see docs/REBASING.md).

diff --git a/components/browsing_data_strings.grdp b/components/browsing_data_strings.grdp
index e176743323fa9ded9a7df9472c2acbcefbbb9e08..4849fa42561c2280eef3bbaa00a2b73fde0abeab 100644
--- a/components/browsing_data_strings.grdp
+++ b/components/browsing_data_strings.grdp
@@ -169,8 +169,8 @@
   <message name="IDS_DEL_COOKIES_COUNTER_ADVANCED_WITH_SIGNED_IN_EXCEPTION" desc="A counter showing the number of sites that use cookies, followed by an exception message that the user will stay signed in to their Google account.">
     {COUNT, plural,
       =0 {None}
-      =1 {From 1 site (you'll stay signed in to your Google Account)}
-      other {From # sites (you'll stay signed in to your Google Account)}}
+      =1 {From 1 site (you'll stay signed in to your MoltBrowser Account)}
+      other {From # sites (you'll stay signed in to your MoltBrowser Account)}}
   </message>
   <message name="IDS_ANDROID_DEL_COOKIES_COUNTER_ADVANCED" desc="A counter showing the number of sites that use cookies on the Clear browsing data screen on Android. It also explains that deleting these cookies will sign you out of most sites.">
     {COUNT, plural,
@@ -216,7 +216,7 @@
     <ph name="TAB_COUNT">$1<ex>5 tabs</ex></ph> from <ph name="WINDOW_COUNT">$2<ex>2 windows</ex></ph> on this device
   </message>
   <message name="IDS_DEL_GOOGLE_COOKIES_SIGNOUT_LINK" desc="A message that will be appended to IDS_DEL_COOKIES_COUNTER_ADVANCED if the user is signed in. It informs the user that their Google cookies will only be deleted if they signed out of Chrome. This string includes a link that, when clicked on, will sign the user out of Chrome.">
-    To delete Google cookies from this device, <ph name="BEGIN_LINK">&lt;a href="#" target="_blank" id="signOutLink"&gt;</ph>sign out of Chrome<ph name="END_LINK">&lt;/a&gt;</ph>.
+    To delete Google cookies from this device, <ph name="BEGIN_LINK">&lt;a href="#" target="_blank" id="signOutLink"&gt;</ph>sign out of MoltBrowser<ph name="END_LINK">&lt;/a&gt;</ph>.
   </message>
   <message name="IDS_SETTINGS_MANAGE_IN_GOOGLE_PASSWORD_MANAGER" desc="Description for the 'Passwords and Passkeys' linkout in the 'Other Google Data' subpage inside the 'Delete browsing data' dialog. This lets users know that passwords and passkeys can be managed and deleted in Google password manager.">
     Manage in Google Password Manager
